src/NUnitTestAdapter/VSTest2NUnitFilterConverter.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace NUnit.VisualStudio.TestAdapter
+{
+    public class VsTest2NUnitFilterConverter
+    {
+        private string NUnitFilter { get; }
+        public VsTest2NUnitFilterConverter(string vstestfilter)
+        {
+            NUnitFilter = vstestfilter
+                .Replace("TestCategory", "Category")
+                .Replace("FullyQualifiedName","test")
+                .Replace("~","=~");
```

Comment:
VS Test filter also support `!~` operator (was added here https://github.com/microsoft/vstest/pull/1803).
Should it be accounted here?